script/Ciel/Day 8/08_01_CIEL08_1B.txt

```diff
@@ -138,27 +159,31 @@ I am in complete agreement. People can lie as much as they want with writings."
 [sha:7efa3eaceb93667851f1f72547ce2a060793572d]{
 -- Page 45, Offset 18374.
 -- 当主さま。まことに言いづらいのですが、爪が猫のものだったら猫なんじゃないかな、その人。
-Dear Matriarch, though it pains me to say, someone with a cat's paws is, in fact, a cat.
+// If the paw fits
+My dear Matriarch... I hate to break it to you, but anyone with cat's paws is, in fact, a cat.
 }
 [sha:12d3b41d3ffacb20caf83212c3e5e1ac749b6952]{
 -- Page 46, Offset 18375.
 -- 「……まあ、そこまでは言ってないけど。
-"...Well, I wouldn't go that far.
+"Well, I wouldn't go that far...
 }
 [sha:e3214b65497c30ea3e3ae066784d3fc880f1ee4f]{
 -- Page 46, Offset 18376.
 -- 簡素な人間関係もそれはそれで便利だと思うよ。知り合う人ぜんぶに全身全霊で向き合ってたら体力が持たないし。
-I think relationships that are kept simple are also useful in their own way. It would be exhausting if I had to completely devote myself when talking to anyone I know.
+I think it's fine for some relationships to be kept simple.
+ It'd be exhausting if I had to bare my heart to every single person I met.
 }
 [sha:b42136e15a7110c594f0275e63c1eace1d942eb4]{
 -- Page 46, Offset 18377.
 -- 色んなところに行けるようになって、いろんな人と知り合えるようになった以上、人間関係も最適化させないと潰れちまう……ってのは友人からの受け売りだけど」
-Because it means that you can go to various places and meet various people, you have to optimize your relationships or you'll be crushed. That's what a friend told me anyway."
+Once you've been all sorts of places and met with all sorts of people,
+ you have to start optimizing your relationships or you'll end up crushed.
+ That's what a good friend told me, at any rate."
```

Comment:
> That's what a good friend _once_ told me, at any rate.

Feels like it's missing the once with this expression

Reply:
I feel like with the "at any rate", the "once" makes the sentence too heavy. I prefer current version